From e8000ab6142175b63ba2c8365837f5c1d60bbae0 Mon Sep 17 00:00:00 2001
From: Otavio Pontes <otavio.pontes@intel.com>
Date: Tue, 5 May 2020 12:52:48 -0700
Subject: [PATCH] config: CONFIG_FILE_PATH shouldn't be treated as a path

CONFIG_FILE_PATH contains a list of paths not just one path. So
we shouldn't use realpath to point to it.

Signed-off-by: Otavio Pontes <otavio.pontes@intel.com>
---
 src/globals.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/globals.c b/src/globals.c
index 60f17aa7..5bb33918 100644
--- a/src/globals.c
+++ b/src/globals.c
@@ -739,7 +739,7 @@ static bool load_flags_in_config(char *command, struct option *opts_array, const
 	struct stat st;
 	char *ctx = NULL;
 	char *config_file = NULL;
-	char *str = realpath(CONFIG_FILE_PATH, NULL);
+	char *str = strdup_or_die(CONFIG_FILE_PATH);
 	char *full_command = NULL;
 
 	if (!str) {
-- 
2.26.2

